src/components/JoinUsBar/JoinUsBar.css

```diff
@@ -0,0 +1,34 @@
+
+/*Problema da risolvere: centrare e ordine il bottone e il testo*/
+.join_us_bar {
+    background-color: #fc3f1a; /*#120068;*/
+    height: 156px;
+    justify-content:center;
+    align-items: center;
+    text-align: center;
+}
+
+.JSB_button {
+  background-color: white; /*#fc3f1a;*/
+  color: black;
+  padding: 0.4em 5em;
+  border-radius: 1em;
+  text-align: center;
+  cursor: pointer;
+  margin-top: 0.5em;
+}
+
+.JSB_button:active {
```

Comment:
Prova con :hover, come suggeriva nick, in caso avessi voglia è interessante la differenza tra active e hover

Reply:
Interessante la differenza tra active/focus/hover, grazie per il suggerimento :) 
In hover ora modifico solo l'opacità.